src/wallet/subcommands/wallet_cmd.rs

```diff
@@ -319,7 +319,8 @@ impl WalletCommands {
             Self::New { signature_type } => {
                 let signature_type = match signature_type.to_lowercase().as_str() {
                     "secp256k1" => SignatureType::Secp256k1,
-                    _ => SignatureType::Bls,
+                    "bls" => SignatureType::Bls,
+                    _ => SignatureType::Delegated,
```

Comment:
```
forest-wallet new blah
t410fqxtk3l42yvbfvkkna4ai4gx2ot7ck7rh2owreoy
```
This should be an error.

Reply:
Actually, earlier we were using `_ => SignatureType::Bls`, that's why I didn't change, anways I will update it.

Reply:
Yes, I know. I think it will less error prone.

Reply:
`WalletCommands::New` subcommand is currently taking a String; maybe it's time to move to an enum.

Reply:
Yeah that would be much better. I will update it.